rollup.config.js

```diff
@@ -64,7 +64,7 @@ export default {
   },
   // NOTE - the order matters for the extensions below
   plugins: [
-    resolve({ extensions, browser: true }),
+    resolve({ extensions, browser: true, preferBuiltins: true }),
```

Comment:
The default value is `true`, just suppressing log messages that come up when this value is not explicitly set.